job-descriptions/account-executive.md

```diff
@@ -14,32 +14,37 @@ To create a product that serves the needs of all developers, we are building a d
 
 If you are passionate about making the world better through software, come join us!
 
-### About the role
+## About the role
 
 - Own the relationships with our customers and prospects, which include many of the largest and most innovative organizations in the world
 - Develop new opportunities inside existing customers
 - Build the playbook for educating and guiding customers to successful adoption of Sourcegraph
 - Work with our product team to be the voice of the customer and bring market insights to our roadmap
 - Deliver customer feedback to our teammates, including our engineers (who are the most customer-driven engineers you'll ever see--they care about making customers happy and closing deals!)
 
-#### Your qualifications
+### Qualifications
 
 - 5+ years mid-market/enterprise SaaS sales experience
 - Experience selling products with ASP >= \$100k
 - Proficiency with software development concepts
 - Passionate about our mission and the potential of developers to improve the world
 
-#### Nice to haves
+### Nice-to-haves
 
 - Selling developer products and working closely with developers
 - Sales management experience (e.g., small group or geography)
 - Converting freemium usage to large commercial contracts
 - Experience with complex multi-year contracts
 - Proficiency with security, licensing, and compliance requirements
 
-### Compensation and benefits
+## Location
```

Comment:
@dadlerj  - the AE job description didn't have a location section. Added it here and included the blurb from the head of customer success for travel/US calls - want to make sure this is accurate for you!